frontend/package.json

```diff
@@ -31,6 +31,7 @@
     "dependencies": {
         "@hookform/resolvers": "3.3.4",
         "@reduxjs/toolkit": "2.1.0",
+        "@tailwindcss/typography": "0.5.10",
```

Comment:
looks like we don't need this plugin - https://tailwindcss.com/docs/plugins#typography

Reply:
I've read the documentation and this article - https://www.themes.dev/blog/typographic-defaults-in-tailwind-css/#:~:text=are%20in%20Tailwind.-,Set%20defaults%20for%20content%20from%20a%20CMS,-Working%20with%20utility . As I understood, without this "@tailwindcss/typography" plugin we can't change the default styles of typography using just tailwind because there is no special 'prose' class in it. Also when I removed this plugin and cheked if it works it did not change any styles for text. Or am I missing something?(